node/src/accountant/scanners/scanners_utils.rs

```diff
@@ -219,12 +219,27 @@ pub mod payable_scanner_utils {
         }
     }
 
+    #[derive(Debug, PartialEq, Eq)]
     pub struct PendingPayableTriple<'a> {
```

Comment:
If you want you may call this `PendingPayableMetadata`, in case someone wants to extend this `struct` in the future.